src/main/java/core/basesyntax/CsvReportCreator.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Map;
+
+public class CsvReportCreator implements ReportCreator {
+    private String filePath;
+
+    public CsvReportCreator(String filePath) {
+        this.filePath = filePath;
+    }
+
+    @Override
+    public void createReport(Map<String, Integer> fruitInventory,
+                             String filePath) throws IOException {
```

Comment:
You have exception handling in catch block, there is no sense to also have this exception in method header
```suggestion
    public void createReport(Map<String, Integer> fruitInventory,
                             String filePath) {
```